rpc/params/controller.go

```diff
@@ -61,17 +61,18 @@ type RemoveBlocksArgs struct {
 
 // ModelStatus holds information about the status of a juju model.
 type ModelStatus struct {
-	ModelTag           string                `json:"model-tag"`
-	Life               life.Value            `json:"life"`
-	Type               string                `json:"type"`
-	HostedMachineCount int                   `json:"hosted-machine-count"`
-	ApplicationCount   int                   `json:"application-count"`
-	UnitCount          int                   `json:"unit-count"`
-	OwnerTag           string                `json:"owner-tag"`
-	Machines           []ModelMachineInfo    `json:"machines,omitempty"`
-	Volumes            []ModelVolumeInfo     `json:"volumes,omitempty"`
-	Filesystems        []ModelFilesystemInfo `json:"filesystems,omitempty"`
-	Error              *Error                `json:"error,omitempty"`
+	ModelTag           string                 `json:"model-tag"`
+	Life               life.Value             `json:"life"`
+	Type               string                 `json:"type"`
+	HostedMachineCount int                    `json:"hosted-machine-count"`
+	ApplicationCount   int                    `json:"application-count"`
```

Comment:
I considered this, but decided against suggesting for compatibility.

However, thinking about it it will be fairly trivial to to check in the client, just need to default to `len(Applications)` if `ApplicationCount` doesn't exist